src/hotspot/share/interpreter/zero/bytecodeInterpreter.cpp

```diff
@@ -1371,7 +1371,6 @@ void BytecodeInterpreter::run(interpreterState istate) {
       arrayOop arrObj = (arrayOop)STACK_OBJECT(arrayOff);                      \
       jint     index  = STACK_INT(arrayOff + 1);                               \
       const int add_len = 32;                                                  \
-      STATIC_ASSERT(add_len == strlen("Index  out of bounds for length "));    \
       /* Two integers, the additional message, and the null-terminator */      \
       char message[2 * jintAsStringSize + add_len + 1];                        \
```

Comment:
Sorry for another nit. Since `STATIC_ASSERT` is gone, we can inline `add_len`, and make the whole thing `+ 33`.

Reply:
Fixed. Thanks.